app/controllers/katello/api/v2/sync_plans_controller.rb

```diff
@@ -47,10 +47,6 @@ def show
     def create
       sync_date = sync_plan_params[:sync_date].to_time(:utc)
```

Comment:
it looks like if 'sync_date' isn't passed in at all (and so sync_plan_params[:sync_date]) is nill, the controller still returns a 500 ISE. 

You might change that to:

`sync_date = sync_plan_params[:sync_date].try(:to_time, :utc)`